scripts/operations_tests/operations_tests.py

```diff
@@ -172,6 +177,27 @@ def add_nan(self, image_stack, fraction=0.001):
         data.ravel()[nan_indices] = np.nan
         return image_stack
 
+    def add_flats_and_darks(self, params):
+        for frame_type in ['flat_before', 'dark_before', 'flat_after', 'dark_after']:
+            if frame_type in params:
+                filename_group = FilenameGroup.from_file(config_manager.load_sample)
+                filename_group.find_all_files()
+                filename_group.find_related(getattr(FILE_TYPES, frame_type.upper()))
```

Comment:
```
related_filename_group =  filename_group.find_related(getattr(FILE_TYPES, frame_type.upper()))
related_filename_group.find_all_files()
image_stack = loader.load(filename_group=related_filename_group)
```

Reply:
Running tests for Flat-fielding:
No related files found for dark_after
    raise ValueError("selected_flat_fielding not in:", valid_methods)
ValueError: ('selected_flat_fielding not in:', ['Only Before', 'Only After', 'Both, concatenated'])
(mantidimaging-dev) PS C:\Users\44770\PycharmProjects\mantidimaging>